nix_update/eval.py

```diff
@@ -81,7 +81,9 @@ def eval_expression(import_path: str, attr: str, flake: bool) -> str:
 
   position = if pkg ? isRubyGem then
     raw_version_position
-  else
+  else if pkg ? isPhpExtension then
```

Comment:
Where is this attribute defined?

```
$ rg isRubyGem
pkgs/development/ruby-modules/gem/default.nix
249:  passthru = passthru // { isRubyGem = true; };
$ rg isPhpExtension
```

Reply:
Here: https://github.com/NixOS/nixpkgs/pull/207605/commits/6645185b342dfc69efdae296aa0c66929284a202

Reply:
Ok. I will wait for this pull request to hit master than.